docs/css/ghpages-materialize.css

```diff
@@ -1,7 +1,7 @@
+ * Waves v0.7.6
```

Comment:
What was changed here? This makes the review harder.
Can you revert this change?

Reply:
I have no idea too, I'll revert that :+1:

Reply:
reverted : https://github.com/materializecss/materialize/pull/187/commits/ee8dc8c3a99f20cc9b6302cfb0435de6b8877a1c